### PR TITLE
Add listOrphanedDisks script and StorageHelper

### PR DESCRIPTION
**Pull request by michaelandersen**

### CODE REVIEW

**Comment by remibergsma** [comment]
LGTM, merging. Thanks!

File: listOrphanedDisks.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    FORCE = 0
+    global zone
+    zone = ''
+    global clusterarg
+    clusterarg = ''
+    global configProfileName
+    configProfileName = ''
+
+    # Usage message
+    help = "Usage: " + os.path.basename(__file__) + ' [options] ' + \
```
Comment (bechampion) [inline listOrphanedDisks.py]: Have you tried argparse.ArgumentParser?

Comment (bechampion) [inline listOrphanedDisks.py]: I'm literally writiing/wrote the same thing for my work place .. u pass arguments that are meaningful to the CS api such as --zone --name --networkid  ... and i pass those into the call itself ... but i for debugging and others argvs that belong to the application it self i do this sort of thing:  
dict((k, v) for k, v in res.__dict__.items() if v is not None and k !="action" and k !="debug") // to cherry pick argvs that shouldn't land on the CS call

File: cloudstackops/cloudstackops.py [hunk 2]
```diff
@@ -998,6 +1011,37 @@ def getStoragePoolTags(self, toClusterID):
...
+        
+        if volumes is not None:
+            # sort results by domain
+            volumes.sort(key=lambda vol: vol.domain, reverse=True)
+        
+            # select volumes with no vmname attached
+            for volume in volumes:
+                if volume.vmname is None:
+                    orphans.append(volume)
+
```
Comment (bechampion) [inline cloudstackops/cloudstackops.py]: orphans = [x for x in volumes if x.vmname is None] // maybe?

**Pull request merged**